src/MICore/UnixUtilities.cs

```diff
@@ -74,9 +74,16 @@ public static class UnixUtilities
 
         internal static string GetDebuggerCommand(LocalLaunchOptions localOptions)
         {
+            string miDebuggerCommand = localOptions.MIDebuggerPath;
+
+            if (!String.IsNullOrEmpty(localOptions.MIDebuggerArgs))
+            {
+                miDebuggerCommand = String.Concat(miDebuggerCommand, " " + localOptions.MIDebuggerArgs);
+            }
+
```

Comment:
I don't know if this matters much, but by doing this here instead of in `LaunchLocalDebuggerCommand`, you will wind up putting these args in front of `--interpreter=mi` instead of after.

My suggestion: Find all of the spots in MIEngine where we have hard coded the string `--interpreter=mi` and replace it with a call to a helper that gets the args.

Here are the spots that I can see:
  C:\dd\MIEngine\src\MICore\LaunchOptions.cs(135):            string dbgCmdArguments = String.Format(CultureInfo.InvariantCulture, "{0} {1}", dbgPath, "--interpreter=mi");
  C:\dd\MIEngine\src\MICore\LaunchOptions.cs(791):                        startRemoteDebuggerCommand = "gdb --interpreter=mi";
  C:\dd\MIEngine\src\MICore\LaunchOptions.cs(795):                        startRemoteDebuggerCommand = "lldb-mi --interpreter=mi";
  C:\dd\MIEngine\src\MICore\Transports\LocalTransport.cs(28):            proc.StartInfo.Arguments = "--interpreter=mi";
  C:\dd\MIEngine\src\MICore\UnixUtilities.cs(58):                "{4} --interpreter=mi --tty=$DbgTerm < {1} > {2} & " +